lib/storage/tree.rb

```diff
@@ -0,0 +1,78 @@
+#==============================================================================
+# Copyright (C) 2022-present Alces Flight Ltd.
+#
+# This file is part of Flight Storage.
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which is available at
+# <https://www.eclipse.org/legal/epl-2.0>, or alternative license
+# terms made available by Alces Flight Ltd - please direct inquiries
+# about licensing to licensing@alces-flight.com.
+#
+# Flight Storage is distributed in the hope that it will be useful, but
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, EITHER EXPRESS OR
+# IMPLIED INCLUDING, WITHOUT LIMITATION, ANY WARRANTIES OR CONDITIONS
+# OF TITLE, NON-INFRINGEMENT, MERCHANTABILITY OR FITNESS FOR A
+# PARTICULAR PURPOSE. See the Eclipse Public License 2.0 for more
+# details.
+#
+# You should have received a copy of the Eclipse Public License 2.0
+# along with Flight Storage. If not, see:
+#
+# For more information on Flight Storage, please visit:
+# https://github.com/openflighthpc/flight-storage
+#==============================================================================
+require "tty-tree"
+
+module Storage
+  class Tree
+    attr_reader :name, :children
+  
+    def initialize(name, children)
+      @name = name
+      @children = children
+    end
+    
+    # Convert to a format tty-tree can render properly
+    def toTTY

```

Comment:
Please use `snake_case` for method names (https://github.com/rubocop/ruby-style-guide#snake-case-symbols-methods-vars)

Also, the name of this method is misleading. If I were to call `Tree#to_tty`, I would expect it to return an instance of `TTY::Tree`; what I would get is a hash of form `{ 'name' => [...] }`. Please could you either:
- rename this method
- modify its return value to be an instance of `TTY::Tree` (I think this makes more sense)